.gitignore

```diff
@@ -4,3 +4,6 @@ lib/*/*
 coverage.info
 npm-debug.log
 phantomjsdriver.log
+package-lock.json
+
+.idea
```

Comment:
please remove the `.idea` as it's not related to joint js, you can put this record into excludes instead (`.git/info/exclude`)